code/modules/surgery/organs/organ_internal.dm

```diff
@@ -34,9 +34,11 @@
 	for(var/X in actions)
 		var/datum/action/A = X
 		A.Grant(M)
+	SEND_SIGNAL(src, COMSIG_ORGAN_INSERTED, owner)
 
 //Special is for instant replacement like autosurgeons
 /obj/item/organ/proc/Remove(mob/living/carbon/M, special = FALSE)
+	SEND_SIGNAL(src, COMSIG_ORGAN_REMOVED, owner)
```

Comment:
Why are you passing in the owner when that can be retrieved from the organ already passed in as the source?

Also when making signals you need to include all arguments that could be useful, not just for your own purpose.
```suggestion
	SEND_SIGNAL(src, COMSIG_ORGAN_REMOVED, M, special)
```